packit/utils.py

```diff
@@ -7,6 +7,8 @@
 
 import git
 
+from packit.exceptions import PackitException
+
 logger = logging.getLogger(__file__)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```
I've been wondering why it's so silent...;-)

Reply:
`git blame` and let's find the person...

Reply:
```
Author:     Frantisek Lachman <flachman@redhat.com>                                                                                                                          
AuthorDate: Thu Feb 14 14:00:02 2019 +0100                                                                                                                                   
Commit:     Frantisek Lachman <flachman@redhat.com>                                                                                                                          
CommitDate: Thu Feb 14 14:00:02 2019 +0100                                                                                                                                   

    Fix sg2srpm

    Signed-off-by: Frantisek Lachman <flachman@redhat.com>   

...

diff --git a/packit/utils.py b/packit/utils.py                                                                                                                               
index dfaa924..67874fb 100644                                                                                                                                                
--- a/packit/utils.py                                                                                                                                                        
+++ b/packit/utils.py                                                                                                                                                        
@@ -7,7 +7,7 @@ import tempfile                                                                                                                                              

 import git

-logger = logging.getLogger("source_git")                                                                                                                                    
+logger = logging.getLogger(__file__)                                                                                                                                        
```
:D

Reply:
Looks like St. Valentines day was not a good day for my code..;-)